style.css

```diff
@@ -635,7 +632,7 @@
     letter-spacing: 0.37px;
     text-align: left;
     margin-bottom: 0.01px;
-    animation: 5s infinite reverse translation linear;
+    animation: 5s reverse translation linear;
```

Comment:
- [ ] - [ OPTIONAL ] Well done @chepkok3, you have done your best on what is asked in the requirement, however, I have a suggestion I would like to raise for you to fix, please take a look back at your record below, your current style is broken, I think if you work on this to make few improvements that will really be a great step in making your website even better, remember that your portfolio is your online representation 🤌 

- That was a genius idea on adding the the back to top button, but I think it would be even better if you style it to look like the design.
- the rest of the desktop version is clear enough to see what is broken and not aligned, pleas take of adjusting all.

![5](https://user-images.githubusercontent.com/77843050/208685345-2efc19ee-932c-4662-a912-6553d32e30da.gif)

Reply:
@joskalenda Thank you for noticing the broken styling for the desktop version 🙏. I will fix it.